VoodooI2C/VoodooI2C/VoodooI2CDevice/VoodooI2CDeviceNub.cpp

```diff
@@ -126,19 +126,19 @@ IOReturn VoodooI2CDeviceNub::getDeviceResourcesDSM(UInt32 index, OSObject **resu
     *result = nullptr;
 
     if (!(availableIndex & (1 << index))) {
-        IOLog("%s::%s TP7G index 0x%x is not supported\n", getName(), acpi_device->getName(), availableIndex);
+        IOLog("%s::%s Returned index 0x%x from _DSM or XDSM method is not supported\n", getName(), acpi_device->getName(), availableIndex);
         return kIOReturnUnsupportedMode;
     }
 
     return evaluateDSM(I2C_DSM_TP7G, index, result);
 }
 
-IOReturn VoodooI2CDeviceNub::getAlternativeInterrupt(VoodooI2CACPICRSParser* crs_parser) {
+IOReturn VoodooI2CDeviceNub::parseResourcesCRS(VoodooI2CACPICRSParser* crs_parser) {
```

Comment:
This can be set up as pass by reference than a pointer. It would ensure that crs_parser will always be present and not NULL.

Reply:
Done